electron/app/components/CheckboxGrid.tsx

```diff
@@ -1,38 +1,48 @@
-import React, { useContext, useEffect, useState } from "react";
+import React, { useContext, useState } from "react";
 import styled, { ThemeContext } from "styled-components";
 import { Checkbox, FormControlLabel } from "@material-ui/core";
 import { ArrowDropDown } from "@material-ui/icons";
+import { useRecoilValue } from "recoil";
 
 import * as atoms from "../recoil/atoms";
+import * as selectors from "../recoil/selectors";
 
 import Filter from "./Filter";
+import NumericFieldFilter from "./NumericFieldFilter";
 
 const GLOBAL_ATOMS = {
   includeLabels: atoms.filterIncludeLabels,
   invertInclude: atoms.filterInvertIncludeLabels,
   includeNoConfidence: atoms.filterLabelIncludeNoConfidence,
   confidenceRange: atoms.filterLabelConfidenceRange,
+  confidenceBounds: selectors.labelConfidenceBounds,
 };
 
 const MODAL_ATOMS = {
   includeLabels: atoms.modalFilterIncludeLabels,
   invertInclude: atoms.modalFilterInvertIncludeLabels,
   includeNoConfidence: atoms.modalFilterLabelIncludeNoConfidence,
   confidenceRange: atoms.modalFilterLabelConfidenceRange,
+  confidenceBounds: selectors.labelConfidenceBounds,
 };
 
 const Body = styled.div`
   vertical-align: middle;
+  font-weight: bold;
 
-  label {
-    width: 100%;
-    height: 32px;
+  & > div {
```

Comment:
This change seems to have increased the spacing between rows, which I think looked better before.

Before:
![image](https://user-images.githubusercontent.com/3719547/92517840-5be8a700-f1e5-11ea-8882-3c8de09c1848.png)

After:
![image](https://user-images.githubusercontent.com/3719547/92517831-555a2f80-f1e5-11ea-9b24-32ebc37b4c8b.png)

Reply:
+1 for more compact

Reply:
Addressed